core/tests/template_model/test_template_model_crud.py

```diff
@@ -0,0 +1,123 @@
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Module for testing template model CRUD."""
+
+from sqlite3 import IntegrityError as Sqlite3IntegrityError
+import datetime
+
+import freezegun
+from django.test import TestCase
+from django.db.utils import IntegrityError as DjangoIntegrityError
+
+from core.models.template_model import TemplateModel
+from core.tests import helper
+
+
+class TemplateModelUpdateTestCase(TestCase):
+    """Test cases for Template model update."""
+
+    @classmethod
+    def setUpTestData(cls):
+        super(TemplateModelUpdateTestCase, cls)
+        cls._template_model_values = helper.build_template_model_dict_data()
+
+    @freezegun.freeze_time('2020-03-10 00:00:00')
+    def setUp(self):
+        super(TemplateModelUpdateTestCase, self).setUp()
+        self._template_model = TemplateModel(**self._template_model_values)
+        self._template_model.save()
+
+    @freezegun.freeze_time('2020-03-11 00:00:00')
+    def test_update_template_model_created_at_auto_now_add(self):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the first line](https://app.codacy.com/manual/lucas625/Template_Django/pullRequest?prid=5220949)